### PR TITLE
Sass 835 - Interest design iteration journey tests

### PR DESCRIPTION
**Pull request by dnyuar**
### Description
Journey test changes for interest design iteration

Add a link to the relevant story in Jira
[SASS-835](https://jira.tools.tax.service.gov.uk/browse/SASS-493)

##### Before Reviewing
 - [x]  Have you pulled the branch down?
 - [x]  Have you assigned yourself to the PR? 
 - [x]  Have you moved the task to “in review” on JIRA? 
 - [x]  Have you checked to ensure all dependencies are up to date? 

##### Whilst Reviewing
 - [x]  Have you run the journey tests?
 - [x]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
 - [ ]  Have you checked for merge conflicts?
 - [ ]  Have you checked to make sure there are no builds in the pipeline before you merge? 
 - [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
 - [x]  Have you run the journey tests? 
 - [x]  Have you addressed warnings where appropriate? 

##### After PRs been Reviewed
 - [ ]  Have you checked for merge conflicts?

### CODE REVIEW

**Comment by matt712** [comment]
Webdriver factory could be bumped to 0.22.0. 
We need to be careful merging this as it will be a breaking change until SASS-833 is ready

**Review by matt712 — APPROVED** [review]

**Comment by matt712** [comment]
Only approving as it matches content changes, still need some stuff done

File: .bsp/sbt.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.4.9","bspVersion":"2.0.0-M5","languages":["scala"],"argv":["/Library/Java/JavaVirtualMachines/jdk1.8.0_241.jdk/Contents/Home/jre/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/dengnyuar/Library/Application Support/JetBrains/IdeaIC2021.1/plugins/Scala/launcher/sbt-launch.jar","xsbt.boot.Boot","-bsp"]}
```
Comment (joeyymoore) [inline .bsp/sbt.json:1]: whats this file?

Comment (dnyuar) [reply .bsp/sbt.json:1]: Not sure actually

Comment (matt712) [reply .bsp/sbt.json:1]: https://scalameta.org/metals/blog/2020/11/06/sbt-BSP-support.html
This article outlines it. I think it is autogenerated

Comment (dnyuar) [reply .bsp/sbt.json]: I've removed the file

**Pull request merged**